blueprints/redirect-upload-requests/blueprint.json

```diff
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
+	"meta": {
+		"title": "Redirect upload requests",
```

Comment:
This title could be clearer:

```suggestion
		"title": "Serve media files from another host",
```

Reply:
I went ahead and updated the title. I'm happy to go back and re-adjust if you like the previous one better @ivanblagdan